mifiel.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
-  spec.required_ruby_version = '~> 2.3'
+  spec.required_ruby_version = '>= 2.3'
```

Comment:
Would be good to add Ruby 3.0 and 3.1 to `.travis.yml` so that the tests run against them, as well.

Reply:
Good catch! will do. Thanks!